modules/persistent-storage-csi-azure-file-nfs.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent_storage-csi-azure-file.adoc
+//
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-azure-file-nfs_{context}"]
+= NFS support
+
+{product-title} 4.14, and later, supports Azure File Container Storage Interface (CSI) Driver Operator with  Network File System (NFS) with the following caveats:
+
+* Azure File with NFS applies a FS group policy at the driver level and pods cannot override it.
```

Comment:
I would say something such as:
Azure File CSI with NFS does not honor the fsGroupChangePolicy requested by pods. Azure File CSI with NFS applies a default OnRootMismatch fsGroup policy regardless of the policy requested by the pod.

I wonder if we shouldn't put that statement inside an "important" red square.
cc @gnufied @bertinatto

Reply:
+1 to updated wording.

Reply:
Fixed.